packages/fake-api/package.json

```diff
@@ -30,11 +30,7 @@
   },
   "devDependencies": {
     "@kumahq/config": "*",
-    "eslint": "^9.22.0",
-    "typescript": "^5.8.2"
-  },
-  "peerDependencies": {
-    "vite": "^6.0.0",
-    "cypress": "^14.0.0"
+    "typescript": "^5.8.2",
+    "vite": "^6.0.0"
```

Comment:
I guess we will probably end up moving these to root also, but lets wait and see with just `eslint` for now